spncreation.ps1

```diff
@@ -15,20 +15,20 @@ param
 
 function Get-AzureCmdletsVersion
 {
-    $module = Get-Module AzureRM -ListAvailable
+    $module = Get-Module Az -ListAvailable
     if($module)
     {
         return ($module).Version
     }
-    return (Get-Module Azure -ListAvailable).Version
+    return (Get-Module Az -ListAvailable).Version
 }
 
 function Get-Password
 {
     $currentAzurePSVersion = Get-AzureCmdletsVersion
-    $azureVersion511 = New-Object System.Version(5, 1, 1)
+    $minAzurePSVersion = New-Object System.Version(5, 6, 0)
```

Comment:
Az PowerShell version 8 is now available, I would recommend considering the most recent version of this module for this script.
```suggestion
    $minAzurePSVersion = New-Object System.Version(8, 0, 0)
```